### PR TITLE
more details for the pnu process

### PR DESCRIPTION
**Pull request by heoelri**
## Purpose

* PR contains more detailed information around the PNU process

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## Other Information

As discussed with @jadarsie and @walterov - please decide if that level of detail makes sense or not.

### CODE REVIEW

**Comment by heoelri** [comment]
@rtibi / @walterov / @jadarsie - this PR contains the changes we've discussed here: https://github.com/Azure/AKSEngineOnStack/pull/44#issuecomment-644974849 I think it's valuable to mention and explain these details and enterprise customers will definitly ask for it. But i leave it up to you to decide if it makes sense here or not. 

If you think that this should be incorporated into [Upgrade a Kubernetes cluster on Azure Stack Hub](https://docs.microsoft.com/en-gb/azure-stack/user/azure-stack-kubernetes-aks-engine-upgrade?view=azs-2002) on docs.microsoft.com, i'm happy to raise a PR there and to either drop this one or keep it here as well.

File: AKSe-on-AzStackHub/README.md [hunk 1]
```diff
@@ -314,14 +314,14 @@ Secrets/credentials should never be stored in plaintext in your application code
...
 
 - [Upgrade to a newer Kubernetes and base OS image version](https://docs.microsoft.com/azure-stack/user/azure-stack-kubernetes-aks-engine-upgrade#steps-to-upgrade-to-a-newer-kubernetes-version)
 - [Upgrade the base OS image only](https://docs.microsoft.com/azure-stack/user/azure-stack-kubernetes-aks-engine-upgrade#steps-to-only-upgrade-the-os-image)
 
-Newer base OS images contain the latest OS security fixes and kernel updates. These images have to be downloaded by the Azure Stack Hub Operator.
+Newer base OS images contain the latest OS security fixes and kernel updates. Updates that were released between two OS image versions or before a newer base OS image version is available in the Azure Stack Hub Marketplace are automatically installed using Debian's (the foundation Ubuntu, which is the Linux distro that is used for the Kubernetes cluster nodes, is build on top of) [Unattended Upgrade](https://wiki.debian.org/UnattendedUpgrades) mechanism to automatically apply the latest security (and other) updates automatically.
```
Comment (jadarsie) [inline AKSe-on-AzStackHub/README.md:322]: Not sure if this makes clear that Unattended Upgrades are enabled by default

File: AKSe-on-AzStackHub/README.md [hunk 2]
```diff
@@ -314,14 +314,14 @@ Secrets/credentials should never be stored in plaintext in your application code
...
+
+The _Unattended Upgrade_ mechanism does not automatically reboot cluster nodes. This can be done automatically using the open-source [kured (KUbernetes REboot Daemon)](https://docs.microsoft.com/en-us/azure/aks/node-updates-kured) that watches for Linux nodes that require a reboot, then automatically handle the rescheduling of running pods and node reboot process.
```
Comment (jadarsie) [inline AKSe-on-AzStackHub/README.md:324]: Not sure if this makes clear that some upgrades wont be applied after a reboot.

**Comment by jadarsie** [comment]
> @rtibi / @walterov / @jadarsie - this PR contains the changes we've discussed here: [Azure/AKSEngineOnStack#44 (comment)](https://github.com/Azure/AKSEngineOnStack/pull/44#issuecomment-644974849) I think it's valuable to mention and explain these details and enterprise customers will definitly ask for it. But i leave it up to you to decide if it makes sense here or not.
> 
> If you think that this should be incorporated into [Upgrade a Kubernetes cluster on Azure Stack Hub](https://docs.microsoft.com/en-gb/azure-stack/user/azure-stack-kubernetes-aks-engine-upgrade?view=azs-2002) on docs.microsoft.com, i'm happy to raise a PR there and to either drop this one or keep it here as well.

Thx @heoelri

File: AKSe-on-AzStackHub/README.md (hunk 1 above)
Comment (heoelri) [reply AKSe-on-AzStackHub/README.md]: i've reworked the section a bit - hope it's clearer now.

File: AKSe-on-AzStackHub/README.md (hunk 2 above)
Comment (heoelri) [reply AKSe-on-AzStackHub/README.md]: reworked the section

**Comment by heoelri** [comment]
@jadarsie i've rewritten the section - i hope it's clearer now.

**Pull request merged**